formatter_test.go

```diff
@@ -38,7 +43,7 @@ func (f F) Format(s fmt.State, c rune) {
 	fmt.Fprintf(s, "F(%d)", int(f))
 }
 
-type Stringer struct { i int }
+type Stringer struct{ i int }
```

Comment:
Thanks for fixing this, but it's unrelated to the struct field tags change, so do you mind leaving it out? I (or someone) can fix this formatting in another pull request.

Reply:
This was from `gofmt -w -l -s`, I can leave how it was no problem.